.travis.yml

```diff
@@ -1,61 +1,51 @@
-# we need trusty so apt/oracle-java8-set-default works
 dist: trusty
 language: python
 python:
-  - "2.7"
+- '2.7'
 sudo: false
 services:
-  - redis-server
+- redis-server
 addons:
-  postgresql: "9.3"
+  postgresql: '9.3'
   apt:
     packages:
-      - oracle-java8-set-default
+    - oracle-java8-set-default
 before_install:
-  - redis-server --version
-  - wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
-  - chmod +x miniconda.sh
-  - ./miniconda.sh -b
-  - export PATH=/home/travis/miniconda3/bin:$PATH
-  # Update conda itself
-  - conda update --yes conda
+- openssl aes-256-cbc -K $encrypted_295d008f0223_key -iv $encrypted_295d008f0223_iv -in .auth.json.enc -out .auth.json -d
```

Comment:
I was assuming we'd use travis secrets?

https://docs.travis-ci.com/user/encryption-keys/

Reply:
It looks like that may not be possible either because the travis documentation lists **Please note that encrypted environment variables are not available for pull requests from forks.**